docs: elaborate on Scylla admin REST API (#11849)
scylladb/scylladb: docs/operating-scylla/rest.rst

REST
------

> Scylla exposes a REST API to retrieve administrative information from a node and execute administrative operations.

Reviewer: ```suggestion
ScyllaDB exposes a REST API to retrieve administrative information from a node and execute administrative operations.```

> Scylla exposes a REST API to retrieve administrative information from a node and execute administrative operations.

Reviewer: ```suggestion
ScyllaDB exposes a REST API to retrieve administrative information from a node and execute administrative operations.
```

> Examples are configuration status and updates, cluster-level information, and more.

Reviewer: ```suggestion
For example, it allows you to check or update configuration, retrieve cluster-level information, and more.
```

> :doc:`Nodetool </operating-scylla/nodetool>` CLI tool interacts, using JMX, with a *scylla-jmx* process, which in turn, uses the REST API to interact with the Scylla process.     

Reviewer: ```suggestion
The :doc:`nodetool </operating-scylla/nodetool>` CLI tool interacts with a *scylla-jmx* process using JMX. The process, in turn, uses the REST API to interact with the ScyllaDB process.
```

> You can interact with the REST API directly using :code:`curl` or using the Swagger UI available at :code:`your-ip:10000/ui`

Reviewer: ```suggestion
You can interact with the REST API directly using :code:`curl` or using the Swagger UI available at :code:`your-ip:10000/ui`.
```

> For example, using the Swagger UI with Scylla Docker:

Reviewer: ```suggestion
The following example shows using the Swagger UI with the Scylla Docker image.
```

Author: @annastuchlik its hard for me to parse this suggestion

.. code:: sh

  docker run --name some-scylla -p 10000:10000 -d scylladb/scylla:5.0.4 --api-address 0.0.0.0

> and visting http://localhost:10000/ui/

Reviewer: ```suggestion
Then visit http://localhost:10000/ui/.
```

> .. warning::
> 
>   Do not expose the REST API externally, as done in the example above, for production systems.
>   It will allow external access to administration-level information and operations on your cluster!
>   By default, the API is bound to an internal IP (127.0.0.1), blocked for external connections.
> 

Reviewer: ```suggestion
```
